src/dodal/devices/i24/pmac.py

```diff
@@ -1,19 +1,78 @@
-from ophyd import Component as Cpt
-from ophyd import (
-    Device,
-    EpicsMotor,
-    EpicsSignal,
-)
+from enum import Enum
 
+from bluesky.protocols import Triggerable
+from ophyd_async.core import AsyncStatus, StandardReadable
+from ophyd_async.core.signal import SignalRW
+from ophyd_async.core.signal_backend import SignalBackend
+from ophyd_async.core.soft_signal_backend import SoftSignalBackend
+from ophyd_async.core.utils import DEFAULT_TIMEOUT
+from ophyd_async.epics.motion import Motor
+from ophyd_async.epics.signal import epics_signal_rw
 
-class PMAC(Device):
+HOME_STR = r"\#1hmz\#2hmz\#3hmz"
+ZERO_STR = "!x0y0z0"
+
+
+class LaserSettings(str, Enum):
+    """PMAC strings to switch laser on and off.
+    Note. On the PMAC, M strings usually have to do with position compare
+    set up.
+    For example, for laser1:
+        Use M712 = 0 if triggering on falling edge.
+        Use M712 = 1 if on rising edge.
+    """
+
+    LASER1ON = " M712=1 M711=1"
+    LASER1OFF = " M712=0 M711=1"
+    LASER2ON = " M812=1 M811=1"
+    LASER2OFF = " M812=0 M811=1"
```

Comment:
Nit: I think some `_` would make these more readable:
```suggestion
    LASER_1_ON = " M712=1 M711=1"
    LASER_1_OFF = " M712=0 M711=1"
    LASER_2_ON = " M812=1 M811=1"
    LASER_2_OFF = " M812=0 M811=1"
```